src/server/graphql.js

```diff
@@ -47,12 +43,10 @@ const resolverMap = {
     parseLiteral(ast) {
       if (ast.kind === Kind.INT) {
         return parseInt(ast.value, 10);
-      } else {
```

Comment:
it’s a linter choice that is mostly about reducing nesting and redundancies, basically if there is a return statement in the if block, the rest of the function is the `else` block whether expressly or not.

If you find this uncomfortably implicit we can just disable the rule and re-run formatting though.